frontend/src/v5/store/tickets/card/ticketsCard.redux.ts

```diff
@@ -59,6 +61,7 @@ export const INITIAL_STATE: ITicketsCardState = {
 	view: TicketsCardViews.List,
 	overrides: null,
 	readOnly: false,
+	unsavedTicket: null,
```

Comment:
You need to revert to this state after a ticket is created otherwise if the user tries to create another ticket of the same template directly afterwards all the fields will be prefilled.

Reply:
I do at [newTicket.component.tsx](https://github.com/3drepo/3drepo.io/pull/4682/files#diff-f102d917f230d34018d1a0f60090970fb2427e2c74401ce19011e8f4411169dd), unless I misunderstood what you're saying. In that case, could you add the reproduction steps?